activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -937,6 +937,24 @@ def build_from
       end
     end
 
+    def build_join(*args)
+      first, *rest = args
+
+      if first.is_a?(String) && rest.present?
+        sanitized = @klass.send(:sanitize_sql, args)
+
+        if sanitized == first
+          # First string doesn't contain any parameter substitution: args is a list of raw SQL
+          # strings.
+          args
```

Comment:
Do we explicitly support passing multiple strings to `joins`? I don't think we need to.